scripts/features/r-base.sh

```diff
@@ -19,6 +19,8 @@ fi
 touch /home/$WSL_USER_NAME/.homestead-features/r-base
 chown -Rf $WSL_USER_NAME:$WSL_USER_GROUP /home/$WSL_USER_NAME/.homestead-features
 
-apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
-add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/'
+wget -qO- https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc | sudo gpg --dearmor -o /etc/apt/keyrings/r-project.gpg
```

Comment:
switch to `curl` for consistency?

Reply:
This was weird so I reverted to docs' recommendation.